osu.Game/Modes/UI/StandardHUDOverlay.cs

```diff
@@ -1,26 +1,21 @@
 ﻿// Copyright (c) 2007-2017 ppy Pty Ltd <contact@ppy.sh>.

```

Comment:
Yeeeeeeeah knew this was gonna happen, hence why I said "too late". One of the most painful things to fix in git.